src/__experimental__/components/checkbox/checkbox.component.js

```diff
@@ -97,7 +108,14 @@ Checkbox.propTypes = {
   /** Allows component to be focused on page load */
   autoFocus: PropTypes.bool,
   /** The content for the help tooltip, to appear next to the Label */
-  labelHelp: PropTypes.node
+  labelHelp: PropTypes.node,
+  /** Margin bottom, given number will be multiplied by base spacing unit (8) */
+  mb: PropTypes.oneOf([0, 1, 2, 3, 4, 5, 7]),
+  /** Margin left, any valid CSS value */
+  ml: PropTypes.string,
+  /** Adaptive spacing (spacing changes to align with other input components at a breakpoint).
+  True for default breakpoint (960px) or number for custom breakpoint */
+  adaptiveSpacing: PropTypes.oneOfType([PropTypes.bool, PropTypes.number])
```

Comment:
have you thought to separate that into two props to avoid confusion? e.g.:
```
adaptiveSpacing: PropTypes.bool,
customBreakpoint: PropTypes.number
```

Reply:
I wanted to avoid adding two new props for it, I don't really think it's confusing like that is it? The prop description makes it pretty clear I think! Happy to discuss it though 👍